### DISCUSSION TITLE
Embedding Tableau dashboard in quarto blog

### DISCUSSION THREAD

**Discussion by sigolyori** [quarto-dev/quarto-cli#3549] [Q&A]
Hi, quarto community!

I want to embed the Tableau dashboard using HTML embed code in Jupyter notebook.
Is there any way to embed native HTML code in Jupyter notebook and render it with Quarto?

Below is my embed code format.

In advance, thanks for the excellent tool for blogging! It makes my life easier.

`<div class='tableauPlaceholder' id='viz1669955106756' style='position: relative'><noscript><a href='#'><img alt='공간정보 탐색적 분석 대회 ' src='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;je&#47;jeju_story&#47;sheet14&#47;1_rss.png' style='border: none' /></a></noscript><object class='tableauViz'  style='display:none;'><param name='host_url' value='https%3A%2F%2Fpublic.tableau.com%2F' /> <param name='embed_code_version' value='3' /> <param name='site_root' value='' /><param name='name' value='jeju_story&#47;sheet14' /><param name='tabs' value='no' /><param name='toolbar' value='yes' /><param name='static_image' value='https:&#47;&#47;public.tableau.com&#47;static&#47;images&#47;je&#47;jeju_story&#47;sheet14&#47;1.png' /> <param name='animate_transition' value='yes' /><param name='display_static_image' value='yes' /><param name='display_spinner' value='yes' /><param name='display_overlay' value='yes' /><param name='display_count' value='yes' /></object></div>                <script type='text/javascript'>                    var divElement = document.getElementById('viz1669955106756');                    var vizElement = divElement.getElementsByTagName('object')[0];                    vizElement.style.width='100%';vizElement.style.height=(divElement.offsetWidth*0.75)+'px';                    var scriptElement = document.createElement('script');                    scriptElement.src = 'https://public.tableau.com/javascripts/api/viz_v1.js';                    vizElement.parentNode.insertBefore(scriptElement, vizElement);                </script>`

**Comment by dragonstyle** [comment]
You should be able to emit raw HTML by surrounding the HTML with:

````
```{=html}

<html here>

```
````

That will make a raw block which should pass through the HTML untouched.

**Reply by sigolyori** [reply]
It works beautifully. Thank you!

**Comment by mcanouil** [comment]
FYI, Quarto Dashboards are available in the v1.4 pre-release: https://quarto.org/docs/dashboards/
